src/Ignitor/HttpServer/index.ts

```diff
@@ -102,13 +102,17 @@ export class HttpServer {
     })
 
     this.server.instance!.on('error', async (error: NodeJS.ErrnoException) => {
+      this.application.logger.error(JSON.stringify(error))
+
       if (error.code === 'EADDRINUSE') {
         this.application.logger.error('Port in use, closing server')
         process.exitCode = 1
-        return
+      } else if (error.code === 'ENOTFOUND') {
+        this.application.logger.error('Host not found, closing server')
+        process.exitCode = 1
+      } else {
+        await this.kill(3000)
```

Comment:
I believe this should stay out of the `if/else` block

Reply:
This may be stepping beyond the bounds of a PR from someone outside the project, but I removed the `return` from the conditional and moved the `kill` into an explicit `else` block rather than adding another `return` since returning within a conditional and leaving outside the conditional as the implicit `else` can obscure the flow.